File: Algorithm-Toolbox/week1/sum.cpp
#include <iostream>

int main() {
    int firstElement;
    int secondElement;
    int sum;

    std::cin >> firstElement >> secondElement; 

    sum = firstElement + secondElement;

    std::cout << "" << firstElement 
                    << " + "
                    << secondElement
                    << " = "
                    << sum 
                    << std::endl;
    
    return 0;
}

File: Algorithm-Toolbox/week1/Sum.java
import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner inputUser = new Scanner(System.in);
    Integer firstElement = inputUser.nextInt();
    Integer secondElement = inputUser.nextInt();

    Integer sum = firstElement + secondElement;

    System.out.println("" + firstElement + " + " + secondElement + " = " + sum);
  }
}